src/metrics/metrics.md

```diff
@@ -1,11 +1,62 @@
 # Metrics
 
-Here we provide definitions and information about our standard metrics.
+Here we provide definitions and information about our standard metrics.  We use some terminology that is defined [here](https://docs.telemetry.mozilla.org/concepts/terminology.html).
 
-Table of contents with links to all metrics.
+## Table of Contents
+
+<!-- toc -->
+
+-----
+
+## Daily Active Users (DAU)
+
+### Overview
+
+Daily Active Users or DAU counts the number of unique profiles active in the product on each day.  This is intended to approximate the number of people using the product each day.
+
+### Details
+
+The concept of a "unique profile" may vary from product to product and the relationship between that concept and actual users may also vary.  As such care should be taken interpreting this metric as an actual count of users.
```

Comment:
What is the reader supposed to take away from this paragraph? "Don't think that you'll understand what this metric is by the time you've finished reading the document - you're going to have to hunt elsewhere to get unspecified extra knowledge"? Can we do some of the legwork for the reader and be more useful?

Reply:
Perhaps we can just link to the "profile vs users" section as we did below. 

Suggested revision:

```suggestion
DAU counts unique profiles. Keep in mind that a [profile is not necessarily a user](/concepts/analysis_gotchas.html#profiles-vs-users)
```

Reply:
What was the resolution here?